src/main/java/com/google/sps/api/calendar/CalendarInterface.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.sps.api.calendar;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.CalendarListEntry;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.Events;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.api.authorization.AuthorizationRequester;
+import java.io.IOException;
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CalendarInterface implements Serializable {
+  public static final String PRIMARY_CALENDAR_FLAG = "primary";
+  // Default timezone
+  public static final String CET_TIME_ZONE = "Europe/Zurich";
```

Comment:
Nit: I'd fallback to UTC instead. Not because it might be used by more people but because you would notice the error much earlier if the fallback was used suddenly and your events would be generated 2h early.

Reply:
also, it might then make sense to call the variable `DEFAULT_TIME_ZONE` instead because the variable shouldn't depend on its own value. That would just make it unnecessarily hard to change the value.

Reply:
Then let's fall back to Sydney time?

Reply:
I was going to also argue that UTC is more intuitive because it's 0 offset. But I guess in the end it doesn't really matter that much